.github/workflows/build-and-test.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name: Compile project
         run: cabal build --enable-tests
       - run: cabal test --enable-tests
-      - run: cabal haddock | grep "100%" | wc -l | grep -G "[45]" # Fixes issue with different haddock coverage with different ghc versions https://github.com/haskell/haddock/issues/123
+      - run: cabal haddock -v
```

Comment:
Hey @DavidMazarro That will help to know the results from `cabal haddock`. However, the purpose of this task was to ensure that the library always provides enough documentation (at least 4 to 5 modules with a 100% of documentation). We should preserve that.

Reply:
Ah, I didn't know that, and this made me realize exactly what was the issue with the CI pipeline in #174: there are 6 modules with 100% documentation, and this is looking for 4 or 5 exactly. We can preserve that check, but we would need to change it somehow, IMO.